test/lib/redis-store-spec.js

```diff
@@ -60,6 +60,22 @@ describe ('initialization', function () {
 });
 
 describe('set', function () {
+  it('should return a promise', function (done) {
+    assert.ok(redisCache.set('foo', 'bar') instanceof Promise);
+    done();
+  });
+
+  it('should resolve promise on success', function (done) {
+    redisCache.set('foo', 'bar').then(result => {
+      assert.equal(result, 'OK');
+      done();
+    });
+  });
+
+  it('should reject promise on error', function (done) {
+    redisCache.set('foo', null).catch(() => done());
```

Comment:
I can't `catch(done)` because then `err` gets passed to `done()` and the test will fail. It's equivalent to `catch(err => done(err))`.